torch/multiprocessing/reductions.py

```diff
@@ -160,6 +131,3 @@ def init_reductions():
 
     for t in torch._tensor_classes:
         ForkingPickler.register(t, reduce_tensor)
-
-    ForkingPickler.register(Variable, reduce_variable)
-    ForkingPickler.register(Parameter, reduce_variable)
```

Comment:
I don't think it's a good idea to get rid of this all. It should remove the grad.